1. Comments and Documentation.md

```diff
@@ -1 +1,46 @@
-## 1 Izohlar & Sharhlar
+## 12 Izohlar
+
+### 12.0 Kirish
+Izohlar faqat kodni tushuntirish uchun ishlatiladi. Kod bo'lsa ham bo'lmasa ham. Kod yo'q joylarga sharhlar qanday yoziladi? Kelajakda yozishingiz mumkin bo'lgan kodlarga izoh yozib ketasiz va keginchalik izohlarni ostida kodlaringizni yozib ketishingiz mumkin.
+
+### 12.1 Mualliflik huquqi
+Izohlarni ta'kidlashda quyidagi qoidaga amal qilish kerak:
+
+##### Tavsiya etiladi
+```csharp
+    // ---------------------------------------------------------------
+    // Copyright (c) Coalition of the Good-Hearted Engineers
+    // FREE TO USE TO CONNECT THE WORLD
+    // ---------------------------------------------------------------
+```
+
+##### Tavsiya etilmaydi
+```csharp
+
+    //----------------------------------------------------------------
+    // <copyright file="StudentService.cs" company="OpenSource">
+    //      Copyright (C) Coalition of the Good-Hearted Engineers
+    // </copyright>
+    //----------------------------------------------------------------
+
+```
+
+##### Bu holatda ham tavsiya etilmaydi
+```csharp
+   /* 
+    * ==============================================================
+    * Copyright (c) Coalition of the Good-Hearted Engineers
+    * FREE TO USE TO CONNECT THE WORLD
+    * ==============================================================
+    */
+```
+
+### 12.2 Funksiyalar
+
+Dasturni ishlab chiqish vaqtida mavjud bo'lmagan yoki murakkab funksiyalarni bajaradigan kodda quyidagi ma'lumotlar hisobga olingan bo'lishi kerak.
+
+- Maqsad
+- Daromadlar
+- Natijalar
```

Comment:
Income/Outcome => Kiruvchi/Chiquvchi Ma'lumot ma'nosida kelyapti

Reply:
aha to'g'rilaymiz